src/api/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_07_27_110437) do
+ActiveRecord::Schema[7.0].define(version: 202302224114624) do
```

Comment:
This timestamp is a bit weird, February 2023?

Reply:
Ah nice catch. This is reverted